README.md

```diff
@@ -1 +1,160 @@
-# Metrics-webapp
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Country Info Webapp <a name="about-project"></a>
+
+**Country Info Webapp** is an application that allows users to access the details of various countries that have been populated on the homepage.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+- JavaScript
+- REACT.
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://www.javascript.com/">React</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+<br>
+
+- **React-Based SPA**
+- **Up-to-Date Country stats**
+- **Effortless Country Search**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] As mentioned in the previous review, It requested in the project requirement that the project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/). And we should add the link to the online version of the app to the README file.
![Screenshot from 2023-09-14 11-25-16](https://github.com/iankiema/Country-Info-webapp/assets/91142432/5ed2c278-37f2-4d9d-b57f-23dc15972912)

Reply:
- Project deployment complete